docs/components/connectors/out-of-the-box-connectors/github.md

```diff
@@ -86,6 +86,9 @@ The **GitHub Connector** currently supports the following operations.
 #### Search issue
 
 - **GitHub API:** [Search issue](https://docs.github.com/en/rest/search?apiVersion=2022-11-28#search-issues-and-pull-requests).
+
+<!--- I assume we are referencing the Camunda 8 REST API below, but can we confirm? --->
```

Comment:
CC @sbuettner

Reply:
Its the GitHub API as its the GitHub Connector.